### ISSUE TITLE
mesh scale not supported

### ISSUE THREAD

**Issue by benaxelrod** [personalrobotics/or_urdf#6]
My mesh files were created in mm, and therefore i have specified the scale in the urdf:

```xml
<geometry>
  <mesh filename="package://lca_description/meshes/link1.dae" scale="0.001 0.001 0.001" />
</geometry>
```
However, when I load the urdf into OpenRave with this plugin, the meshes are huge.  

I see where urdf_loader.cpp hard-codes the scale to 1.0.  I might take a shot at making a patch, but I don't have much time to spend on it.
Thanks.

**Comment by benaxelrod** [comment]
Well, it looks like OpenRave doesn't set the scale properly.  I made this modification to line 388 of urdf_loader.cpp with no luck.  

```c++
            printf("scale: %f %f %f\n", mesh.scale.x, mesh.scale.y, mesh.scale.z);
            geom_info->_vRenderScale = OpenRAVE::Vector(mesh.scale.x, mesh.scale.y, mesh.scale.z);
```
The scale is being read properly from the urdf.  But the mesh files are still huge.

**Comment by mkoval** [comment]
That is the correct fix for the render geometry. It's a bit more complicated to fix the collision geometry because OpenRAVE seems to completely ignore the `_vCollisionScale` property. I ended up manually scaling the collision mesh to work around this issue. See #7.

Is it possible that you weren't seeing any effect because you were viewing the collision geometry? (most likely due to #5)

**Comment by mkoval** [comment]
Fixed in #7.
